src/main/resources/static/_shared.yml

```diff
@@ -3197,7 +3197,7 @@ components:
         propertyName: service
         mapping:
           CAS1: '#/components/schemas/ApprovedPremisesUser'
-          CAS3: '#/components/schemas/TemporaryAccommodationUser'
+          temporary-accommodation: '#/components/schemas/TemporaryAccommodationUser'
```

Comment:
is there a particular reason we've done this? we should be using the CAS naming where possible